zebra/zebra_vty.c

```diff
@@ -720,13 +719,13 @@ vty_show_ip_route_detail (struct vty *vty, struct route_node *rn, int mcast)
 	  vty_out (vty, " ago%s", VTY_NEWLINE);
 	}
 
-      for (ALL_NEXTHOPS_RO(re->nexthop, nexthop, tnexthop, recursing))
-	{
+      for (ALL_NEXTHOPS_RO(re->nexthop, nexthop))
+    {
```

Comment:
I would say indentation was already broken (supposed to be GNU style, but it's not), but ok, I'll try to _fix_ all three of them.